pkg/util/log/setup/log.go

```diff
@@ -117,6 +116,33 @@ func SetupLogger(loggerName LoggerName, logLevel, logFile, syslogURI string, sys
 	scrubber.AddStrippedKeys(mergeAdditionalKeysToScrubber(
 		flareStrippedKeys,
 		cfg.GetStringSlice("scrubber.additional_keys")))
+
+	// Registering a callback in case of "log_level" update
+	cfg.OnUpdate(func(setting string, oldValue, newValue any) {
+		if setting != "log_level" || oldValue == newValue {
+			return
+		}
+		level := newValue.(string)
+
+		seelogLogLevel, err := log.ValidateLogLevel(level)
+		if err != nil {
+			return
```

Comment:
Since we can't return it here, would it be worth logging an errir in the various `if err != nil` of this func?

Reply:
Good idea ! Done in 3bf2ae77434cc6e601b74f33ae125377a7f3ab89.